src/events/components/SearchModule/index.tsx

```diff
@@ -28,41 +29,63 @@ const SearchModule: FC = () => {
       )
     );
 
+  const handleToDateClick = (day: Date) => {
+    const clonedDate: Date = new Date(day.getTime());
+    const datetime = DateTime.fromMillis(clonedDate.setHours(0, 0, 0, 0));
+    const dateStartDateTime = DateTime.fromISO(dateStart || DEFAULT_DATE_START_PARAM);
+
+    if (datetime > dateStartDateTime || datetime.toISODate() === dateStartDateTime.toISODate()) {
+      setDateEnd(datetime.toISODate());
+    }
+  };
+
+  const handleFromDateClick = (day: Date) => {
+    const clonedDate: Date = new Date(day.getTime());
+    const datetime = DateTime.fromMillis(clonedDate.setHours(0, 0, 0, 0));
+    const dateEndDateTime = DateTime.fromISO(dateEnd || DEFAULT_DATE_END_PARAM);
+
+    if (datetime < dateEndDateTime || datetime.toISODate() === dateEndDateTime.toISODate()) {
+      setDateStart(datetime.toISODate());
+    }
+  };
+
   return (
-    <div className={style.grid}>
-      <input
-        className={style.searchInput}
-        type="search"
-        defaultValue={search || DEFAULT_SEARCH_PARAM}
-        placeholder="Søk"
-        onChange={(event) => setSearch(event.target.value)}
-      />
-      <DateInput
-        label="Fra: "
-        time={DateTime.fromISO(dateStart || DEFAULT_DATE_START_PARAM).toFormat('yyyy-MM-dd')}
-        onChange={(event) => setDateStart(event.target.value)}
-      />
-      <DateInput
-        label="Til: "
-        time={DateTime.fromISO(dateEnd || DEFAULT_DATE_END_PARAM).toFormat('yyyy-MM-dd')}
-        onChange={(event) => setDateEnd(event.target.value)}
-      />
-      <SelectMultiple
-        onEventTypesInput={onEventTypesInput}
-        eventTypes={JSON.parse(eventTypes || DEFAULT_EVENT_TYPES_PARAM)}
-      />
-      <label>
-        Vis arrangementer med påmelding
-        <ToggleSwitch
-          onChange={() =>
-            setAttendanceEventsChecked(
-              attendanceEventsChecked === 'true' || attendanceEventsChecked === null ? 'false' : 'true'
-            )
-          }
-          checked={attendanceEventsChecked === 'true' || attendanceEventsChecked === null}
+    <>
+      <div className={style.grid}>
+        <input
```

Comment:
This could probably be wrapped as a general component?

Reply:
I think that is outside the scope of this PR